packages/good-design/src/stories/apps/goodid/GoodId.stories.tsx

```diff
@@ -77,7 +77,7 @@ export const OnboardScreenExample = () => {
 
 export const OffersAgreementExample = () => (
   <Wizard>
-    <OffersAgreement />
+    <OffersAgreement width={375} />
```

Comment:
why are you using fixed width here?
we should be able to test in stories how components expand

Reply:
forgot to turn this into a storybook component with arguments, done now (for all stories)